src/audio/host.c

```diff
@@ -68,8 +68,8 @@ struct host_data {
 	uint32_t copy_blocking;	/**< True for copy in blocking mode */
 
 	/* local and host DMA buffer info */
-#if !CONFIG_DMA_GW
 	struct hc_buf host;
+#if !CONFIG_DMA_GW
 	struct hc_buf local;
```

Comment:
Here we still branch on  CONFIG_DMA_GW. Is this as intended? Because the PR description says we no longer need to branch on this config option.

Reply:
@tlauda I now see the last patch where the config is removed. So, it looks good to me.